.clang-format.full

```diff
@@ -0,0 +1,115 @@
+---
+BasedOnStyle: LLVM
+Language: Cpp
+Standard: Auto
+
+IndentWidth: 2
+UseTab: Never
+ColumnLimit: 100
```

Comment:
I'm not a huge fan of hard limits on line sizes, but I could probably live with it.

Reply:
I prefer it to be able to tile and see multiple files next to each other without the horrors of line-wrapping.